tools/devtool

```diff
@@ -582,9 +582,18 @@ apply_linux_61_tweaks() {
 
 
 apply_performance_tweaks() {
-  # Disable turbo boost. Some of our tests are performance tests, and we want minimum variability wrt processor frequency
-  # See also https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/processor_state_control.html
-  sudo sh -c "echo 1 > /sys/devices/system/cpu/intel_pstate/no_turbo" &> /dev/null
+  # m6a instances do not support the amd_pstate driver (yet), so nothing we can do there
+  if [[ -d /sys/devices/system/cpu/intel_pstate ]]; then
```

Comment:
Should we try to do that for other drivers as well?

Reply:
Afaik for gravitron there's nothing to do (cpu frequency is constant anyway), and on AMD we don't have the pstate driver yet (https://wiki.archlinux.org/title/CPU_frequency_scaling#amd_pstate). I can have a look at acpi_pstate for AMD instances real quick if that allows us to do anything!

Reply:
Seems like we can at least disable turbo boost for m6a.metal, so done that :)

Reply:
awesome, thanks :)